### PR TITLE
Refs #22338 - add worker_timeout param

### PR DESCRIPTION
**Pull request by chris1984**

### CODE REVIEW

**Comment by chris1984** [comment]
Tests are failing because Pulp does not have the param yet, once that is merged should pass.

File: manifests/init.pp [hunk 1]
```diff
@@ -66,6 +66,10 @@
 #
 # $manage_broker::                      Manage the qpid message broker when applicable
 #
+# $pulp_worker_timeout::                The amount of time (in seconds) before considering a worker as missing. If Pulp's
+#                                       mongo database has slow I/O, then setting a higher number may resolve issues where workers are
+#                                       going missing incorrectly. Defaults to 30.
```
Comment (ekohl) [inline manifests/init.pp]: This should match params.pp

**Comment by chris1984** [comment]
@ekohl tests are passing with the merge of https://github.com/theforeman/puppet-pulp/pull/299 can I get another review please?

**Comment by ekohl** [comment]
Thanks!

**Pull request merged**